### PR TITLE
C/fix/code improvement

### PR DESCRIPTION
**Pull request by UserXGnu**
As we all know, Virgulino's c version was passing through a heavy change on it's design. It's done now.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,7 @@ DEST=virgulino
 DESTDIR=/usr/bin/
 CC=gcc
 SWITCH=-o
+CVERSION=-std=c11
```
Comment (gjuniioor) [inline Makefile]: You add CVERSION variable but don't use this.

File: src/output.h [hunk 2]
```diff
@@ -17,17 +21,18 @@ void help (void);
...
-    strncat (msg, err, 100-strlen(msg));
+    char msg[128];
+    sprintf (msg, "[%s-%s] ", T_RED, NOTHING);
+    strncat (msg, err, 128-strlen(msg));
 
-    fprintf (stderr, "%s\n", msg);
+    fprintf (stderr, "%s ...\n", msg);
 }
+
 void
```
Comment (gjuniioor) [inline src/output.h]: What you think about remove this function `splash`? In my opinion it is needless.

Comment (UserXGnu) [inline src/output.h]: I think that it's good once the we don't use it anyway.

File: Makefile (hunk 1 above)
Comment (UserXGnu) [inline Makefile]: I've created this variable so that we can decide if we'll use it or not!

File: src/output.h (hunk 2 above)
Comment (gjuniioor) [inline src/output.h]: Nice, so, remove :)

File: Makefile (hunk 1 above)
Comment (gjuniioor) [inline Makefile]: We need test in a old compiler. I'll try do this.

Comment (gjuniioor) [inline Makefile]: Sorry this delay but I tested in `gcc 4.8.4` and using `-std=c11` work fine. Just some warnings, but compiled the source and executed fine too.

**Comment by UserXGnu** [comment]
Done as requested. CVERSION is now included on compilation process.

**Comment by gjuniioor** [comment]
Merge it, dude :)

**Pull request closed without merging**